### DISCUSSION TITLE
Agricraft create config inside config

### DISCUSSION THREAD

**Discussion by ferreiraalex** [ACGaming/UniversalTweaks#272] [Bugfix Suggestions]
i have bug and try to find fix and found this
https://github.com/AgriCraft/AgriCraft/issues/1208
but for me i have this bug with groovy mod
idk what do this but if you can find please fix this
![explorer_pDcJqdCkgo](https://github.com/ACGaming/UniversalTweaks/assets/10839855/a20b9e63-9776-4929-b4e2-bb91b4f7fe9f)
after every game start creates new inside folder

**Comment by ACGaming** [comment]
Fixed in CleanroomMC/GroovyScript@e66914548604ff232091d2eedda2769418f73427
